torch/csrc/jit/interpreter.cpp

```diff
@@ -736,6 +736,35 @@ Operation getOperation(jit::Node* node, bool values_are_variables) {
       return 0;
     };
   IR_ELSE()
+    switch (node->kind()) {
```

Comment:
note: this is not pretty but it is the easiest thing we can do right now to make  operators added directly by onnx work. Once size tracing gets merged and tested these can be removed.

Reply:
Why do we need them though? Why are we interpreting ONNX ops?